assets/js/editor-components/feedback-prompt/index.tsx

```diff
@@ -34,33 +37,32 @@ const FeedbackPrompt = ( {
 	}, [] );
 
 	return (
-		isVisible && (
-			<div className="wc-block-feedback-prompt">
-				<Icon icon={ commentContent } />
-				<h2 className="wc-block-feedback-prompt__title">{ title }</h2>
-				<p className="wc-block-feedback-prompt__text">{ text }</p>
-				<a
-					href={ url }
-					className="wc-block-feedback-prompt__link"
-					rel="noreferrer noopener"
-					target="_blank"
-				>
-					{ __(
-						'Give us your feedback.',
-						'woo-gutenberg-products-block'
-					) }
-					<Icon icon={ external } size={ 16 } />
-				</a>
-			</div>
-		)
+		<>
```

Comment:
I'm curious about this change. Is there a need to add a fragment here (`<>`)? Or could we leave it as it was?

Reply:
Oh, never mind, I understand it's because we need to return a JSX component.